pkg/envoy/admin.go

```diff
@@ -45,6 +45,9 @@ func DrainListeners(adminPort uint32, inboundonly bool) error {
 	}
 	res, err := doEnvoyPost(drainURL, "", "", adminPort)
 	log.Debugf("Drain listener endpoint response : %s", res.String())
+	if err != nil {
```

Comment:
Not sure what this is for, seems we always return err on L51 anyways?

Reply:
I thought i removed this. Let me clean this up